code/modules/mob/living/living.dm

```diff
@@ -534,6 +534,14 @@
 	if(should_update_mobility)
 		update_mobility()
 
+/mob/living/Crossed(atom/movable/AM)
+	. = ..()
+	for(var/i in get_equipped_items())
+		var/obj/item/item = i
+		item.Crossed(AM)
```

Comment:
As we talked about you should make a new proc for this hook, WearerCrossed() or something

Reply:
Oh I didn't see that part. One second